sc-memory/sc-memory/tests/sc-memory/units/events/test_sc_event.cpp

```diff
@@ -1367,3 +1367,52 @@ TEST_F(ScEventTest, BlockEventsGuardAndEmitAfter)
   std::this_thread::sleep_for(std::chrono::milliseconds(10));
   EXPECT_TRUE(isCalled);
 }
+
+TEST_F(ScEventTest, TwoSubscriptionsForOneArcErasure)
+{
+  ScAddr nodeAddr1 = m_ctx->GenerateNode(ScType::ConstNode);
+  bool isDelayedCalled = false;
+  auto delayedSubscription =
```

Comment:
It seems to me that this is not a delayed subscription, but simply a subscription whose occurrence of events is secondary to instantSubscription

Reply:
renamed them to longExecuted and shortExecuted because first one executes for a longer period of time and second one executes instantly